cmd/controller/main.go

```diff
@@ -153,6 +155,14 @@ func main() {
 		DynamicClient:   dynamicClient,
 	}
 
+	if *maximumPlatformApiVersion != "" {
```

Comment:
Can you extract this to a function that is available at buildpodGenerator initialization?